CONTRIBUTING.md

```diff
@@ -30,6 +30,8 @@ $ rubocop -V
 * Write [good commit messages][3].
 * Use the same coding conventions as the rest of the project.
 * Commit and push until you are happy with your contribution.
+* If you are fixing a bug, prefix the commit message with `[Fix #github-issue-number]`.
```

Comment:
This doesn't apply only to bugs. It applies to anything with an open github issue.